mlflow/statsmodels.py

```diff
@@ -324,8 +327,10 @@ def load_model(model_uri, dst_path=None):
 
     :return: A statsmodels model (an instance of `statsmodels.base.model.Results`_).
     """
-    local_model_path = _download_artifact_from_uri(artifact_uri=model_uri, output_path=dst_path)
-    flavor_conf = _get_flavor_configuration(model_path=local_model_path, flavor_name=FLAVOR_NAME)
+    local_model_path = _download_artifact_from_uri(
+        artifact_uri=model_uri, output_path=dst_path)
```

Comment:
This can be fixed this by commenting `autoformat`.

Reply:
or even better, I can read the Contribution guide more carefully and follow the instructions with precommit hooks too :) - works great!